pulp_container/app/serializers.py

```diff
@@ -105,7 +105,7 @@ class ManifestSerializer(SingleArtifactContentSerializer):
     )
 
     class Meta:
-        fields = SingleArtifactContentSerializer.Meta.fields + (
+        fields = NoArtifactContentSerializer.Meta.fields + (
```

Comment:
With this, users can see artifacts connected to specific manifests, right? With new content, the field will be empty, with old content, the field will reference an artifact.

```
http :5001/pulp/api/v3/content/container/manifests/
```
![image](https://github.com/pulp/pulp_container/assets/8740962/ec53a0e0-bf10-4864-b1ef-f22926a71497)

Reply:
That's a good catch! I will change it back, but put the backward compatibility TODO comment there

Reply:
@lubosmj Nevermind, there seems to be a problem. There's an error in setup when I apply your suggested changes, and if I also change the parent class to SingleArtifactContentSerializer, the unit tests fail.